Code/Editor/EditorViewportSettings.h

```diff
@@ -169,4 +171,7 @@ namespace SandboxEditor
 
     SANDBOX_API float CameraDefaultFarPlaneDistance();
     SANDBOX_API void SetCameraDefaultFarPlaneDistance(float distance);
+
+    SANDBOX_API float CameraDefaultFov();
+    SANDBOX_API void SetCameraDefaultFov(float fov);
```

Comment:
update: It would be great in the parameter name and/or with a comment to specify the value is in radians (also do we want it in radians as otherwise this could be more confusing for authoring fov values 🤔) So perhaps we want to have degrees/radians versions that do the correct mapping and actually store the value in the Settings Registry file as degrees?